### ISSUE TITLE
UWP ListView Scrolling 

### ISSUE THREAD

**Issue by db-007** [Microsoft/WinAppDriver#279]
How do I scroll a ListView in an UWP app.
I am finding the ListView using :class_name like `listview=$SupSession.find_elements(:class_name, "ListView")` but how do I scroll up or down.

I was trying to  `listview[0].send_keys(:page_up)` or `listview[0].send_keys(:page_down)` but that was not doing any scrolling.  It appears that it only moved the cursor only the first issued query but it would not scroll the list view.

Similarly, I trued using `:arrow_down` and `:arrow_up` but that would not do anything either.

I also tried using `:home` and `:end` and that works but it is useless as I still cannot scroll through the list in a scenario "A**nd I scroll down to see 'my-item'**"

**Comment by khouzam** [comment]
The behavior will depend on the application implementation. If you replicate the page up/page down, while interacting directly with the application yourself?

**Comment by db-007** [comment]
@khouzam yes, I can use page up/page down if I interact with my app directly.  But doing it through send_keys as I described above wont work.  And I found the reason :)
It is that I am using irb repl shell to query for elements and when it is used, the irb command will execute but when doing so, it will focus on the application being tested, then focus back into irb shell.  That looses focus on the app being tested.
That explains also why my first send_keys(:page_down) worked but none worked afterwards.

Here is summary:
1. Open app to test
2. open irb shell
3. issue `listview[0].send_keys(:page_down)` in irb shell //this will move focus onto the app, scroll to the last visible item in my ListView, then focus back onto irb shell (**and this is what created my problem**)
4. issue `listview[0].send_keys(:page_down)` again to scroll further into the list is not working anymore

Doing same from a set of feature scenarios script (where no irb shell is involved) shows that scrolling is working.  So, this issue can be closed.  
Much appreciated @khouzam
